Dockerfile.release

```
@@ -0,0 +1,12 @@
+FROM alpine:3.16.0
+
+EXPOSE 8444
```

Should we expose the `8123` port perhaps? 

For the reference: https://github.com/0xPolygon/cdk-validium-node/blob/5f0320c0958e62ffc22b0edfa26a61d0a5dcfc22/Dockerfile#L18

```suggestion
EXPOSE 8123
```